packages/components/src/DateTimePickers/InputDateTimePicker/date-extraction.js

```diff
@@ -0,0 +1,169 @@
+/*
```

Comment:
module comment does not match what the module does and expose

Reply:
I remove that